openmmtools/mcmc.py

```diff
@@ -1143,17 +1150,21 @@ def __getstate__(self):
         serialization = super(LangevinDynamicsMove, self).__getstate__()
         serialization['timestep'] = self.timestep
         serialization['collision_rate'] = self.collision_rate
+        serialization['constraint_tolerance'] = self.constraint_tolerance
```

Comment:
Do we need to add a default key here so that way someone can deserialize an older file without getting a key error here?

Reply:
Yes, good catch! Thanks.